root_profiler/engine/campaign.class.php

```diff
@@ -47,6 +47,10 @@ function Campaign($id = 0)
         $this->_valid = true;
       }
     }
+    function Campaign($id = 0)
+    {
+        self::__construct();
+    }
```

Comment:
Is there any reason other than keeping PHP4 compatibility for keeping old-style constuctors around?

Reply:
Nope. This was intentionally a "least disrupion possible" push.

Reply:
Then I'd say just get rid of them. I don't think anyone sane should be running PHP4 anymore.